gatling-http/src/test/scala/io/gatling/http/request/builder/HttpRequestBuilderSpec.scala

```diff
@@ -92,6 +92,28 @@ class HttpRequestBuilderSpec extends BaseSpec with ValidationValues with EmptySe
       .succeeded shouldBe Seq("BAR")
   }
 
+  it should "work when passing formParamMap" in {
+    val form = Map(
+      "foo" -> "FOO",
+      "baz" -> "BAZ"
+    )
+
+    val session = sessionBase.setAll(form)
+
+    val params = httpRequestDef(
+      _.formParamMap(
+        Map(
+          "${foo}" -> "${baz}"
```

Comment:
Please also test the other cases:
* when the key is a simple non- ELString
* when the value a simple non- ELString
* when the value is not a String